doc/workflow/coding_standard_qml.md

```diff
@@ -0,0 +1,175 @@
+# Coding Standard QML
+
+###1. QML document structure
+
+#####Rule 1.1. Basic document and elements structure should be order like this:
```

Comment:
I mean, that is a matter for discussion. But anyway any comment in code review should be followed by fix or discussion (thats why I did not approved request). We probaly need to add a new best practive for that in workflow.